js-exercises/telephone-check/telephoneCheck.js

```diff
@@ -0,0 +1,9 @@
+function telephoneCheck(...args) {
+  var PhoneNumber = args.toString()
```

Comment:
Don't use 'var' as it will be global. Here you could have used const phoneNumber as you won't be modifying the same.